qt/scientific_interfaces/Muon/ALCBaselineModellingPresenter.h

```diff
@@ -42,6 +43,9 @@ class MANTIDQT_MUONINTERFACE_DLL ALCBaselineModellingPresenter : IALCBaselineMod
   /// Called when on of section selectors is modified
   void onSectionSelectorModified(int index) override;
 
+  /// Subscribe to updates on changes to the corrected data.
+  void subscribe(IALCBaselineModellingPresenterSubscriber *subscriber);
```

Comment:
Just asking your opinion, would it be better to rename this function as `subscribeToInterface` or `subscribeInterface`?

Reply:
I think given the `subscribe` function is always called on the presenter, it provides enough context so can stay as is.